samples/sdl_joystick_gamepad/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+
+#define ARRAY_SIZE(x) (sizeof(x)/sizeof(x[0]))
+
+int main() {
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  SDL_GameController *joy;
+  SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER);
+
+  joy = SDL_GameControllerOpen(0);
+
+  SDL_Event event;
```

Comment:
Again: should be moved to where it's actually used or moved near the top.

It can be part of the `while` loop (recommended / personal preference).